src/Sulu/Component/Media/Tests/Unit/SmartContent/MediaDataProviderTest.php

```diff
@@ -106,6 +111,63 @@ public function testGetConfiguration()
         $this->assertTrue($configuration->hasAudienceTargeting());
     }
 
+    public function testGetTypesConfiguration()
+    {
+        /** @var EntityManagerInterface|ObjectProphecy $entityManager */
+        $entityManager = $this->prophesize(EntityManagerInterface::class);
+
+        /** @var ObjectRepository|ObjectProphecy $objectRepository */
+        $objectRepository = $this->prophesize(ObjectRepository::class);
```

Comment:
Wouldn't it make more sense to call this `$mediaTypeRepository`? `$objectRepository` could be anything.